### PR TITLE
skip tests that use float128 on ppc64le ...

### PR DESCRIPTION
**Pull request by kif**
one test is still failing. Investigation is needed here.

### CODE REVIEW

**Comment by kif** [comment]
close #1244 

Beside this the >f8 attribute becomes a f16 and fails ...

File: h5py/tests/old/test_dataset.py [hunk 1]
```diff
@@ -91,6 +95,8 @@ def test_missing_shape(self):
     def test_long_double(self):
         """ Confirm that the default dtype is float """
         dset = self.f.create_dataset('foo', (63,), dtype=np.longdouble)
+        if platform.machine() in ['ppc64le']:
+            pytest.xfail("Storage of long double deactivated on %s"%platform.machine())
```
Comment (scopatz) [inline h5py/tests/old/test_dataset.py]: Should this be a skip, rather than a fail?

**Comment by kif** [comment]
On Thu, 20 Jun 2019 08:48:40 -0700
Anthony Scopatz <notifications@github.com> wrote:

> Should this be a skip, rather than a fail?

This is up to you ...

**Comment by takluyver** [comment]
I'd say xfail if this is failing now but we hope to fix it and re-enable the tests. Skipping it implies that it can't work, or we don't aim to support it.

**Comment by scopatz** [comment]
Makes sense! It looks like there is one failing style issue https://travis-ci.org/h5py/h5py/jobs/548242859#L323

**Comment by scopatz** [comment]
Thanks @kif!

**Pull request merged**